sdk/cosmosdb/cosmos/test/public/functional/item.spec.ts

```diff
@@ -883,3 +883,197 @@ describe("patch operations", function () {
     });
   });
 });
+
+describe("Item CRUD with priority", function (this: Suite) {
+  this.timeout(process.env.MOCHA_TIMEOUT || 10000);
+  beforeEach(async function () {
+    await removeAllDatabases();
+  });
+  const documentCRUDTest = async function (isUpsertTest: boolean): Promise<void> {
+    // create database
+    const database = await getTestDatabase("sample 中文 database");
+    // create container
+    const { resource: containerdef } = await database.containers.create({ id: "sample container" });
+    const container: Container = database.container(containerdef.id);
+
+    // read items
+    const { resources: items } = await container.items.readAll().fetchAll();
+    assert(Array.isArray(items), "Value should be an array");
+
+    // create an item
+    const beforeCreateDocumentsCount = items.length;
+    const itemDefinition: TestItem = {
+      name: "sample document",
+      foo: "bar",
+      key: "value",
+      replace: "new property",
+    };
+    try {
+      await createOrUpsertItem(
+        container,
+        itemDefinition,
+        { disableAutomaticIdGeneration: true, priorityLevel: "Low" },
+        isUpsertTest
+      );
+      assert.fail("id generation disabled must throw with invalid id");
+    } catch (err: any) {
+      assert(
+        err !== undefined,
+        "should throw an error because automatic id generation is disabled"
+      );
+    }
+    const { resource: document } = await createOrUpsertItem(
```

Comment:
For the sake of maintaining consistency, let's utilize "item" instead of "document" in the JS SDK.

Reply:
In this test "document" is being used in multiple tests. Let me know if this needs to be changed